modules/rosa-kubernetes-concept.adoc

```diff
@@ -9,9 +9,13 @@
 
 {product-title} (ROSA) uses Red Hat OpenShift, which is an enterprise Kubernetes platform. Kubernetes is an open source platform for managing containerized workloads and services across multiple hosts, and offers management tools for deploying, automating, monitoring, and scaling containerized apps with minimal to no manual intervention. For complete information about Kubernetes, see the link:https://kubernetes.io/docs/home/?path=users&persona=app-developer&level=foundational[Kubernetes documentation].
 
-Cluster, compute pool, and compute node:: A Kubernetes cluster consists of a control plane and one or more compute nodes. Compute nodes are organized into compute pools of the type or profile of CPU, memory, operating system, attached disks, and other properties. The compute nodes correspond to the Kubernetes `Node` resource, and are managed by a Kubernetes control plane that centrally controls and monitors all Kubernetes resources in the cluster.
+Cluster and nodes:: A Kubernetes cluster uses nodes to ensure the resilience and scalability of the applications managed on the cluster. Nodes are physical or virtual computing machines that provide resources for the cluster. Kubernetes organizes nodes into control plane nodes and compute plane nodes to support cluster operations. 
 +
-When you deploy the resources for a containerized app, the Kubernetes control plane decides which compute node to deploy those resources on, accounting for the deployment requirements and available capacity in the cluster. Kubernetes resources include services, deployments, and pods.
+The control plane nodes centrally control and monitor all resources in the cluster. When you deploy the resources for a containerized app, the Kubernetes control plane decides which compute plane node to deploy those resources on, accounting for the deployment requirements and available capacity in the cluster. Kubernetes resources include services, deployments, and pods.
```

Comment:
I'd suggest to use https://kubernetes.io/docs/concepts/overview/components/ to stay close to technically accurate definition for these terms.